source/chapters/djing_with_mixxx.rst

```diff
@@ -369,8 +366,17 @@ choose to create them in the preferences.
 Record your mix to disk
 -----------------------
 
-.. versionadded:: 2.0
-   Displays the duration of the recording.
+.. versionadded:: 2.10
```

Comment:
2.1.0 ?  We aren't to 2.10 ... yet! :)

Reply:
IMO these "version added" notices clutter the manual. They're good for release announcements, but I don't know why someone reading the manual would care when a feature was added.